### PR TITLE
Fix sampler aggregation DSL

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

File: src/QueryBuilder/DSL/Aggregation.php [hunk 1]
```diff
@@ -422,13 +422,12 @@ public function adjacency_matrix(string $name): AdjacencyMatrix
...
-    /** sampler aggregation.
+    /**
+     * sampler aggregation.
      *
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-sampler-aggregation.html
-     *
-     * @param string $name
      */
-    public function sampler($name): Sampler
+    public function sampler(string $name): Sampler
```
Comment (deguif) [inline src/QueryBuilder/DSL/Aggregation.php:430]: BC break as type-hint is changed, but this is currently breaking when calling the constructor with another type-hint as constructor only accepts a `string` for name argument.

**Comment by ruflin** [comment]
Could you add a changelog entry? We could consider this a bugfix :-)

**Comment by deguif** [comment]
@ruflin changelog added, can be merged ;)

**Comment by deguif** [comment]
Merging as I added the changelog entry ;)

**Comment by ruflin** [comment]
Thanks!

**Comment by ruflin** [comment]
BTW glad to see you used squash merge to keep the history clean.

**Pull request merged**